server/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM php:7.2-apache
+
+RUN apt-get update \
+    && apt-get install -y \
+       libpng-dev \
+       libpq-dev \
+       libsqlite3-dev \
+       libxml2-dev \
+       libzip-dev \ 
+       git \
+       sqlite \
+       libfreetype6-dev \
+       libjpeg62-turbo-dev \
+    && apt-get autoremove -y \
+    && apt-get autoclean \
+    && apt-get clean \
+    && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/* \
+    && docker-php-ext-install -j$(nproc) \
+       opcache \
+       posix \
+       simplexml \
+       zip \
+    && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
+    && docker-php-ext-install -j$(nproc) gd 
+
+WORKDIR /var/www/html
+
+RUN git clone https://github.com/nextcloud/server.git nc
+RUN cd nc && git submodule update --init
+RUN chown -R www-data:www-data nc
+RUN chsh -s /bin/bash www-data
```

Comment:
Any need for this? You could also simply run the process in the container as root ... there isn't anything other in this container anyways ;)

Reply:
Roeland said that it was not working if /nc is not owned by apache user

Reply:
But if you run apache as root it is owned by it ;) At least that is what works totally fine over here ;)